action/deployment/add.go

```diff
@@ -5,15 +5,32 @@
 package deployment
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/go-vela/cli/internal/output"
 
 	"github.com/go-vela/sdk-go/vela"
 
 	"github.com/go-vela/types/library"
+	"github.com/go-vela/types/raw"
 
 	"github.com/sirupsen/logrus"
 )
 
+// parseKeyValue converts the slice of key=value into a map
+func parseKeyValue(input []string) (raw.StringSliceMap, error) {
+	payload := raw.StringSliceMap{}
+	for _, i := range input {
+		parts := strings.SplitN(i, "=", 2)
+		if len(parts) != 2 {
```

Comment:
Could this be remediated? Either a variable or a no-lint directive?

Reply:
Done.